mlflow_faculty/trackingstore.py

```diff
@@ -167,7 +167,17 @@ def get_run(self, run_uuid):
         :return: A single :py:class:`mlflow.entities.Run` object if it exists,
             otherwise raises an exception
         """
-        raise NotImplementedError()
+        try:
+            faculty_run = self._client.get_run(self._project_id, run_uuid)
```

Comment:
As discussed, should this add the dashes back into the UUID?

Andrew figured out you could do this:

```py
UUID('c867ecce4cce45b1adbbbe17500c71a9')
# => UUID('c867ecce-4cce-45b1-adbb-be17500c71a9')
```

Reply:
Yes, I was gonna do it on a different branch, but I can do it here.